packages/client/test/get.spec.js

```diff
@@ -1,14 +1,15 @@
 /* eslint-env mocha */
 import * as assert from 'uvu/assert'
-import { Web3Storage } from 'web3.storage'
+import { Web3Storage, createRateLimiter } from 'web3.storage'
 
 describe('get', () => {
   const { AUTH_TOKEN, API_PORT } = process.env
   const token = AUTH_TOKEN || 'good'
   const endpoint = new URL(API_PORT ? `http://localhost:${API_PORT}` : '')
+  const rateLimiter = createRateLimiter(Infinity, 1000)
 
   it('get a CAR', async () => {
-    const client = new Web3Storage({ token, endpoint })
+    const client = new Web3Storage({ token, endpoint, rateLimiter })
```

Comment:
How come this is this being set in every test? Each new instance should receive a new limiter and 30 requests in 10 seconds is likely enough for every test.

Reply:
Ah, yes, I added this because the test suite was taking more than twice as long to run with the rate limiting added. My thinking was: add it to every test so that we ensure we're not being unwittingly bitten by the rate limiter, and so that we don't _become_ unwittingly bitten by it when we extend a test to make more API cals. But actually it's only this ['adds big files' test](https://github.com/web3-storage/web3.storage/blob/3dc51309d76360dbb6b764ec4df87a5990841aee/packages/client/test/put.spec.js#L112-L147) which is being limited. So I'll just leave it on that one and remove the rest.

Reply:
Done